libmamba/src/api/channel_loader.cpp

```diff
@@ -86,6 +86,63 @@ namespace mamba
         }
     }
 
+    std::string get_system_certificate()
```

Comment:
shouldn't this return `std::optional<fs::path>`?

Reply:
That would require some change in powerloader; otherwise, we would have to forward a missing value as an empty path. Also I'm not usre of what happens if we don't explicitly set `CURLOPT_CAINFO` to null when the certificate string is empty.